<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueBase</class>
 <widget class="QWidget" name="QueueBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>532</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <layout class="QGridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="pbWhatsThis">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="spacer5_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;font size=&quot;+1&quot;&gt;Packages in Installation Queue&lt;/font&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line1_2">
       <property name="frameShape">
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QueueListView" name="queueView"/>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <spacer name="spacer17">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>16</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSkipHousekeeping">
             <property name="text">
              <string>Skip Housekeeping</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbRemove">
             <property name="text">
              <string>Remove package from Q&amp;ueue after installation</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line5">
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <property name="margin">
          <number>5</number>
         </property>
         <item row="1" column="0">
          <widget class="QPushButton" name="pbClear">
           <property name="maximumSize">
            <size>
             <width>32767</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove all packages in Queue</string>
           </property>
           <property name="text">
            <string>Remove &amp;All</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pbRemove">
           <property name="maximumSize">
            <size>
             <width>32767</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove selected package from Queue</string>
           </property>
           <property name="text">
            <string>&amp;Remove</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pbAdvanced">
           <property name="toolTip">
            <string>Show package details...</string>
           </property>
           <property name="text">
            <string>Details...</string>
           </property>
           <property name="shortcut">
            <string>Return, Ctrl+S</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="spacer4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>200</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="queueFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>620</width>
       <height>176</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QGridLayout">
        <item row="1" column="0">
         <widget class="QPushButton" name="pbCheck">
          <property name="toolTip">
           <string>Check packages in Queue with emerge pretend</string>
          </property>
          <property name="whatsThis">
           <string>Since many applications depend on each other, any attempt to install a certain software package might result in the installation of several dependencies as well. Don't worry, Portage handles dependencies well. If you want to find out what Portage would install when you ask it to install a certain package.</string>
          </property>
          <property name="text">
           <string>Step &amp;1: Check Installation</string>
          </property>
          <property name="shortcut">
           <string>Alt+1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pbGo">
          <property name="toolTip">
           <string>Start emerge of all packages in Queue</string>
          </property>
          <property name="text">
           <string>Step &amp;2: Start Installation</string>
          </property>
          <property name="shortcut">
           <string>Alt+2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <spacer name="spacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>46</width>
            <height>50</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTextBrowser" name="queueBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>90</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="margin" stdset="0">
           <number>2</number>
          </property>
          <property name="text" stdset="0">
           <string>&lt;b&gt;Summary&lt;/b&gt;&lt;br&gt;
&lt;font color=&quot;gray&quot;&gt;Installation Queue is empty.&lt;/font&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;b&gt;Emerge Options&lt;/b&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbUpdate">
          <property name="text">
           <string>Use &amp;Update flag on emerge calls (-uN)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbDownload">
          <property name="text">
           <string>Download all packa&amp;ges before installing them (--fetch-all-uri)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbForceConf">
          <property name="text">
           <string>Force reinstall of config files (--&amp;noconfmem)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbNoWorld">
          <property name="text">
           <string>Do not add the pac&amp;kages to the world profile (--oneshot)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbBackupPkg">
          <property name="text">
           <string>&amp;Backup packages before merging.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>QueueListView</class>
   <extends>QTreeView</extends>
   <header>queuelistview.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kpushbutton.h</include>
  <include location="local">queuelistview.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">ktextbrowser.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
